source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -450,121 +450,6 @@ The above example configures the CDN route to have a default time-to-live of 15
 
 Cookie headers are forwarded to your app by default, so cookie-based authentication will work as expected. Other headers, such as HTTP auth, are stripped by default. If you need a different configuration, see the guidance on [Forwarding Headers](/deploying_services/use_a_custom_domain/#forwarding-headers).
```

Comment:
Who or what forwards the cookie headers? Is it possible to avoid the passive voice here?

Reply:
The `cdn-route` service is the one acting here. I'll make the change.